canonical-websites/snapcraft.io
# Updates testimonials on home page to longer version.

Fixes https://github.com/CanonicalLtd/snapcraft-design/issues/209

Updates testimonial on home page to longer version with headlines.

Copy doc:
https://docs.google.com/document/d/1V0f27l5R3EfGrLTXTpmr1WTisgPDt9d3LPmFEGRwpPc/edit#heading=h.7rue1mg0jjzq

### QA

- ./run
- go to home page
- it should contain longer testimonials

Demo: http://snapcraft.io-pr-190.run.demo.haus/ 

<img width="1132" alt="screen shot 2017-12-18 at 12 50 05" src="https://user-images.githubusercontent.com/83575/34104894-9556842a-e3f2-11e7-9af3-61e03d0c6d8c.png">
